src/core/thread/mle_router.cpp

```diff
@@ -1865,6 +1867,13 @@ void MleRouter::HandleStateUpdateTimer(void)
                 RemoveNeighbor(mRouters[i]);
             }
         }
+        else if (mRouters[i].GetState() == Neighbor::kStateLinkRequest)
+        {
+            if ((TimerMilli::GetNow() - mRouters[i].GetLastHeard()) >= TimerMilli::SecToMsec(kMaxLinkRequestTimeout))
```

Comment:
`kMaxLinkRequestTimeout` is defined in milliseconds.

Reply:
Fixed, thanks.